ICU's rbbicst.pl writes the current year into the copyright line of rbbirpt.h, and
the checked-in file has the license header added by hand (regexcst.pl writes it).
Write the checked-in header, so the output only depends on the input.

--- a/third_party/icu/source/common/rbbicst.pl
+++ b/third_party/icu/source/common/rbbicst.pl
@@ -337,6 +337,8 @@
     #
 
 
+    print "// © 2016 and later: Unicode, Inc. and others.\n";
+    print "// License & terms of use: http://www.unicode.org/copyright.html\n";
     print "//---------------------------------------------------------------------------------\n";
     print "//\n";
     print "// Generated Header File.  Do not edit by hand.\n";
@@ -345,7 +347,7 @@
     print "//    It is generated by the Perl script \"rbbicst.pl\" from\n";
     print "//    the rule parser state definitions file \"rbbirpt.txt\".\n";
     print "//\n";
-    print "//   Copyright (C) 2002-$year International Business Machines Corporation \n";
+    print "//   Copyright (C) 2002-2016 International Business Machines Corporation \n";
     print "//   and others. All rights reserved.  \n";
     print "//\n";
     print "//---------------------------------------------------------------------------------\n";
